Code breacher update, again...

bruberu opened the pull request:
The item has been moved to susycore.

oliwier509 committed:
Update ElectronicCircuits.groovy

Rick8484 committed:
Guh

Rick8484 committed:
Update en_us.lang

oliwier509 committed:
Merge pull request #1572 from Rick8484/patch-10

Guh

oliwier509 committed:
Reverted military loot nerf

the military base is FINALLY anti-cheeseable
no more lactose intolerance
no more @oliwier509 I'm going to raid the military base while riding skeleton hons
no more babcisranie
we gregging in this town now, and your ass is NOT skipping progression

oliwier509 committed:
Made the code breacher EV rather than HV

this way you have to finish HV instead of just getting to it

oliwier509 committed:
fixed the final rickifiation

oliwier509 committed:
lang fix

oliwier509 committed:
fixed code breacher texture

oliwier509 committed:
final fix

now we wait for susy-core to update

oliwier509 committed:
updated susy core

oliwier509 committed:
reverted hash change in pack.toml

minor miscalculation

oliwier509 committed:
moving code breacher to susy-core

oliwier509 committed:
moving code breacher to susy-core

oliwier509 committed:
Delete resources/gregtech/textures/items/metaitems/code_breacher.png

oliwier509 committed:
Delete resources/gregtech/models/item/metaitems/code_breacher.json

bruberu committed:
feat: update susycore and fix codebreacher recipe

bruberu committed:
fix: wait that's a block

oliwier509 reviewed (approved):
Everything has been tested and reviewed
Works exactly as intended

oliwier509 commented:
actually don't merge yet

oliwier509 committed:
Locked all crates behind code breacher

list of changes:
-removed federation proximity mine from the military base, but the sub structure itself still exists for future use :troll:
-buffed the amount of loot generating items in the military base to pre-nerf levels to account for the hard EV requirement for getting it now
-increased the amount of guards in the storage buildings as punishment for making me work with mixins
